addons/native_dialog/gtk_menu.c

```diff
@@ -116,39 +108,8 @@ static GtkWidget *build_menu_item(ALLEGRO_MENU_ITEM *aitem)
       }
       else {
          /* always create an image menu item, in case the user ever sets an icon */
-         gitem = gtk_image_menu_item_new_with_mnemonic(al_cstr(caption));
-         
-         if (aitem->icon) {
-            const int w = al_get_bitmap_width(aitem->icon), h = al_get_bitmap_height(aitem->icon);
-            const int stride = w * 4;
-            int x, y, i;
-            GdkPixbuf *pixbuf;
-            uint8_t *data = al_malloc(stride * h);
-            
-            if (data) {
-               for (y = 0, i = 0; y < h; ++y) {
-                  for (x = 0; x < w; ++x, i += 4) {
-                     al_unmap_rgba(al_get_pixel(aitem->icon, x, y),
-                        &data[i],
-                        &data[i + 1],
-                        &data[i + 2],
-                        &data[i + 3]
-                     );
-                  }
-               }
-               
-               pixbuf = gdk_pixbuf_new_from_data (data, GDK_COLORSPACE_RGB, TRUE, 8,
-                  w, h, stride, destroy_pixbuf, NULL);
-               
-               aitem->extra2 = gtk_image_new_from_pixbuf(pixbuf);
-               
-               gtk_image_menu_item_set_image(GTK_IMAGE_MENU_ITEM(gitem), aitem->extra2);
-               
-               /* Subtract the main reference. the image still holds a reference, so the
-                * pixbuf won't be destroyed until the image itself is. */
-               g_object_unref(pixbuf);
-            }
-         }
+         gitem = gtk_menu_item_new_with_label(al_cstr(caption));
```

Comment:
This should be `gtk_menu_item_new_with_mnemonic`

Reply:
Oops - new commit pushed to fix this. Thanks!